packages/select/src/ux-select.ts

```diff
@@ -261,15 +264,19 @@ export class UxSelect implements UxInputComponent {
   }
 
   private unsetupListAnchor() {
-    this.listAnchor = null;
     this.winEvents.disposeAll();
   }
 
-  public listAnchor: { x: number | string, y: number | string } | null;
-  private calcAnchorPosition() {
-    const elDim = this.element.getBoundingClientRect();
-    const offsetY = (48 - elDim.height) / 2;
-    this.listAnchor = { x: elDim.left, y: elDim.top - offsetY };
+  private setListAnchorPosition() {
```

Comment:
The "set" position job is actually done by the positioning service, so maybe we need a method name to reflect that, or at least avoid giving wrong impression? I think we can just stop at `updateListAnchorPosition`

Reply:
Good call, I'll change it